samples/csharp/MicrosoftGraphSDK/Submission.cs

```diff
@@ -3,6 +3,7 @@
 
 using Microsoft.Graph.Beta;
 using Microsoft.Graph.Beta.Models;
```

Comment:
the new endpoint is not found because you updated `Microsoft.Graph` package.

change the `using` section to don't use beta